src/bundle.h

```diff
@@ -27,6 +27,26 @@ bool is_installed_bundle(const char *bundle_name);
  */
 int required_by(struct list **reqd_by, const char *bundle_name, struct manifest *mom, int recursion, struct list *exclusions, char *msg);
 
+/**
+ * @brief Creates a tracking file in the tracking directory within the
+ * specified state directory.
+ * If there are no tracked files in that directory (directory is empty
+ * or does not exist), copy the tracking directory at
+ * path_prefix/usr/share/clear/bundles to the tracking directory to
+ * initiate the tracking files.
+ * This function does not return an error code because weird state in this
+ * directory must be handled gracefully whenever encountered.
+ * @param bundle_name The name of the bundle.
+ * @param tracking_dir The path to the tracking directory.
+ */
+void track_bundle(const char *bundle_name, const char *state_dir);
```

Comment:
Did you mean to tracking_dir in second_arg
@param state_dir in line 40 The path to the state directory.

Reply:
ups, no, state_dir is the correct argument, at some point it was the tracking_dir but I changed that, I forgot to update the comment, thanks for catching that, I will update that comment

Reply:
I am checking the comments again and I think it's a bit hard to understand what is expected from this function by the name.
As we have a `is_tracked_bundle()` I would call the standard function as `track_bundle(bundle_name)`. And the other `track_bundle_in_statedir(bundle_name, state_dir)`.
The function we would expect to be used more often is the track_bundle. The other is for one specific use case.

Reply:
sounds good, I'll update that.